spec/index.html

```diff
@@ -1462,58 +1462,37 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
-        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
-        [[RFC8259]].</dd>
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of [[RFC8259]],
+        which also conform to the requirements of [[[RFC7493]]] [[RFC7493]].</dd>
 
       <dt id="JSON-value-space">The <a>value space</a></dt>
-      <dd>
-        is the set of all <a data-lt="string">RDF strings</a> that conform to the
-        <a data-cite="RFC8259#section-2">JSON Grammar</a> as described in
-        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
-        [[RFC8259]], and furthermore comply with the following constraints:
-        <ul>
-          <li>They MUST NOT contain any unnecessary whitespace.</li>
-          <li>Keys in objects MUST be ordered lexicographically.</li>
-          <li>Native Numeric values MUST be serialized according to
-            <a data-cite="ECMASCRIPT#sec-tostring-applied-to-the-number-type">Section&nbsp;7.1.12.1</a>
-            of [[ECMASCRIPT]].</li>
-          <li>Strings MUST be serialized with Unicode code points from <code class="codepoint">U+0000</code>
-            through <code class="codepoint">U+001F</code> using lower case hexadecimal Unicode notation
-            (`\uhhhh`) except for the set of predefined JSON control characters —
-            `U+0008`, `U+0009`, `U+000A`, `U+000C`, and `U+000D` — which SHOULD be
-            serialized as `\b`, `\t`, `\n`, `\f`, and `\r`, respectively. All
-            other Unicode characters SHOULD be serialized "as is", except
-            `U+005C` (`\`) and `U+0022` (`"`), which SHOULD be serialized as
-            `\\` and `\"`, respectively.</li>
-        </ul>
-        <div class="issue">The JSON Canonicalization Scheme (JCS) [[RFC8785]]
-          is an emerging standard for JSON canonicalization. This <code>rdf:JSON</code> specification
-          will likely be updated to require such a canonical representation.
-          Users are cautioned against depending on the lexical representation of
-          literals with the <code>rdf:JSON</code> datatype as RDF literals,
-          as the specifics of serialization may change in a future revision of
-          this document.</div>
-        Despite being defined as a set of strings, this value space is
-        considered distinct from the value space of <code>xsd:string</code>,
-        in order to avoid side effects with existing specifications.
-      </dd>
- 
+      <dd>is the set of
+        <a data-cite="RFC8259#section-7">strings</a>,
+        <a data-cite="RFC8259#section-6">numbers</a>,
+        <a data-cite="RFC8259#section-4">objects</a>,
+        <a data-cite="RFC8259#section-5">arrays</a>, and
+        literal values (`true`, `false`, and `null`) [[RFC8259]].
+
+        Two values are considered equal
+        if they are the same <a data-cite="RFC8259#section-7">string</a>,
+        <a data-cite="RFC8259#section-6">number</a>, or
+        literal value;
```

Comment:
So "\u0020" and " " are different.

Reply:
I believe parsing strings with escape sequences resolves those escapes (as it does in RDF). There is a canonical form when serialized via JCS.

Reply:
If you want a data structure that is some transformation of the result of parsing a JSON string then you have to say what this data structure is and what the transformation is.   This is going to involve, for JSON strings, things like handling surrogate pairs (and unpaired surrogates) as well as handling escape sequences.